lib/command.c

```diff
@@ -42,11 +42,80 @@
 #include "command_graph.h"
 #include "qobj.h"
 #include "defaults.h"
+#include "termtable.h"
 
 DEFINE_MTYPE(LIB, HOST, "Host config")
 DEFINE_MTYPE(LIB, STRVEC, "String vector")
 DEFINE_MTYPE(LIB, COMPLETION, "Completion item")
 
+const char *node_names[] = {
```

Comment:
Can we add some documentation to `enum node_type {...`  to point to this string version of the node names.  Additional question, should this instead be part of `struct cmd_node...` and everyone just auto fills it in?  Those that don't won't get the advantage of the search?